sr_multi_moveit/ur10srh_moveit_config/launch/moveit.rviz

```diff
@@ -430,7 +430,7 @@ Visualization Manager:
         Show Robot Visual: true
         Show Trail: false
         State Display Time: 0.05 s
-        Trajectory Topic: move_group/display_planned_path
+        Trajectory Topic: /move_group/display_planned_path
```

Comment:
Why is it necessary to make the name absolute? Generally this is not recommended.